### PR TITLE
Fix null safety errors across DevTools

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Several DevTools workflows were broken due to errors with type casting and null safety.

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/debugger/variables.dart [hunk 1]
```diff
@@ -58,13 +57,13 @@ class ExpandableVariable extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (variable == null) return const SizedBox();
+    final _variable = variable;
```
Comment (polina-c) [inline packages/devtools_app/lib/src/screens/debugger/variables.dart:60]: Starting local variable with '_' looks confusing for me as normally '_' means visibility that does not apply to local variables. Can you name it variable_  or variableLocal instead?

**Review by polina-c — APPROVED** [review]

File: packages/devtools_app/lib/src/primitives/enum_utils.dart [hunk 2]
```diff
@@ -27,7 +27,8 @@ class EnumUtils<T> {
   final Map<String, T> _lookupTable = {};
   final Map<T, String> _reverseLookupTable = {};
 
-  T? enumEntry(String enumName) => _lookupTable[enumName];
```
Comment (polina-c) [inline packages/devtools_app/lib/src/primitives/enum_utils.dart:30]: All errors in this PR are runtime errors, that were not caught by tests. Would it be right to cover the fixes with tests? May be in a separate CL, to speed up the fix landing.

Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/primitives/enum_utils.dart:30]: added some tests and filed https://github.com/flutter/devtools/issues/3991 to track the meta issue of improving devtools testing

File: packages/devtools_app/lib/src/screens/debugger/variables.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/screens/debugger/variables.dart]: renamed.
